test/stiefel.jl

```diff
@@ -4,6 +4,7 @@ include("utils.jl")
     @testset "Real" begin
         M = Stiefel(3,2)
         @testset "Basics" begin
+            @test repr(M) == "Stiefel(3, 2, ℝ)"
```

Comment:
I think `Stiefel{ℝ}(3, 2)` might be better readable (actually for any field-parametrised manifold) and is still a valid constructor?

Reply:
I prefer the `Stiefel(3, 2, ℝ)` case because 1) it's probably the constructor that will be used more often and 2) it's fairly common for math notation of the field of manifolds to include it inside the parentheses (consider e.g. `SO(n, ℝ)` and `GL(n, ℝ)`)

Reply:
I haven't worked with that much, so I trust your expertise. I felt like `Stiefel{ℝ}` to even be a kind of type (some real-valued Stiefel), so that felt nice in view, too. That that's not the most commonly used constructor is clear. So with your expertise, this is resolved.